processor/src/main/java/com/instagram/common/json/annotation/processor/JsonParserClassData.java

```diff
@@ -43,6 +43,7 @@
   private final boolean mPostprocessingEnabled;
   private final String mValueExtractFormatter;
   private final String mParentInjectedClassName;
+  private final boolean mUseGetters;
 
   public JsonParserClassData(
```

Comment:
does it make sense just to pass the class annotation into `JsonParserClassData`?